specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2023-08-01-preview/examples/extensions_chat_completions.json

```
@@ -0,0 +1,64 @@
+{
+  "operationId": "GetChatCompletionsWithAzureExtensions",
+  "title": "Creates a completion for the provided prompt, parameters and chosen model. Uses Azure OpenAI chat extensions.",
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "api-version": "2023-08-01-preview",
+    "deploymentId": "<deployment-id>",
+    "body": {
+      "messages": [
+        {
+          "role": "system",
+          "content": "you are a helpful assistant that talks like a pirate"
+        },
+        {
+          "role": "user",
+          "content": "can you tell me how to care for a parrot?"
+        }
+      ],
+      "dataSources": [
+        {
+          "type": "azureChatExtensionType",
```

I think we should use a valid enum value here:
```suggestion
          "type": "AzureCognitiveSearch",
```